.github/dependabot.yml

```diff
@@ -20,6 +20,19 @@ updates:
     schedule:
       interval: "weekly"
       day: "friday"
+    ignore:
+      - dependency-name: org.jboss.narayana.jta:narayana-jta
+        versions:
+          # require Java 11
+          - ">= 5.13.0"
```

Comment:
I have never seem this syntax for Maven deps, are you sure it works?

Reply:
I've seen it in a bunch of places (https://github.com/apache/avro/blob/main/.github/dependabot.yml#L34, https://github.com/apache/maven-artifact-transfer/blob/mvn4/.github/dependabot.yml#L27, https://github.com/apache/maven-dependency-plugin/blob/master/.github/dependabot.yml#L26, https://github.com/apache/maven-dependency-tree/blob/master/.github/dependabot.yml#L26)

Reply:
Cool, let's merge this one then and see how it works in practice.
If it works Dependabot should close the PRs you mention at the beginning.